### PR TITLE
Unable to pull config files on IBMi

### PR DESCRIPTION
**Pull request by guidofaecke**
Reminds me on the `glob` issue back on ZF2.

### CODE REVIEW

File: config/config.php [hunk 1]
```diff
@@ -23,10 +23,10 @@
     //   - `*.global.php`
     //   - `local.php`
     //   - `*.local.php`
-    new PhpFileProvider('config/autoload/{{,*.}global,{,*.}local}.php'),
+    new PhpFileProvider(realpath(__DIR__) . '/autoload/{{,*.}global,{,*.}local}.php'),
```
Comment (xtreamwayz) [inline config/config.php]: Adding this breaks in a docker image on windows. Any other solution that would work on all systems?

Comment (xtreamwayz) [reply config/config.php]: Actually it does work. I had to remove `config` from the path.

Comment (guidofaecke) [reply config/config.php]: Good to hear!

**Comment by xtreamwayz** [comment]
Merged into develop branch. Thanx @guidofaecke !

**Pull request closed without merging**